docs/site/content/docs/latest/storage.md

```diff
@@ -0,0 +1,146 @@
+# Create Persistent Volumes with Storage Classes
+
+This topic explains how to use dynamic storage in workload clusters.
+
+## Overview: PersistentVolume, PersistentVolumeClaim, and StorageClass
+
+Within a Kubernetes cluster, `PersistentVolume` (PV) objects provide shared storage for cluster pods that is unaffected by pod lifecycles.
+Storage is provisioned to the PV through a `PersistentVolumeClaim` (PVC) object, which defines how much and how the pod accesses the underlying storage.
+For more information, see [Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) in the Kubernetes documentation.
+
+Cluster administrators can define `StorageClass` objects that let cluster users dynamically create PVC and PV objects with different storage types and rules.
+Tanzu also provides default `StorageClass` objects that let users provision persistent storage in a turnkey environment.
+
+`StorageClass` objects include a `provisioner` field identifying the internal or external service plug-in that provisions PVs, and a `parameters` field that associates the Kubernetes storage class with storage options defined at the infrastructure level, such as VM Storage Policies in vSphere.
+For more information, see [Storage Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/) in the Kubernetes documentation.
+
+## Supported Storage Types
+
+Tanzu Kubernetes Grid supports `StorageClass` objects for different storage types, provisioned by Kubernetes internal ("in-tree") or external ("out-of-tree") plug-ins.
```

Comment:
Should references to "Tanzu Kubernetes Grid" be updated to be "Tanzu Community Edition"?

Reply:
yes thanks, missed that one